src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -624,6 +624,11 @@ case class GoogleExtensionsInitializer(cloudExtensions: GoogleExtensions, google
       )
     )
 
-    cloudExtensions.onBoot(samApplication)
+    for {
+      _ <- googleGroupSynchronizer.init()
+      onBootResult <- cloudExtensions.onBoot(samApplication)
+    } yield {
+      onBootResult
+    }
```

Comment:
need to call `googleGroupSynchronizer.init()` here, as opposed to inside a constructor for `GoogleGroupSynchronizer`, because at the time `GoogleGroupSynchronizer` is constructed the db schema has not yet necessarily been created.